examples/google-app-engine/gae-jdk7/src/main/java/com/example/appengine/helloworld/HelloServlet.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2017, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.appengine.helloworld;
+
+import static io.grpc.testing.integration.Messages.SimpleRequest;
+import static io.grpc.testing.integration.Messages.SimpleResponse;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.testing.integration.TestServiceGrpc;
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * This servlet communicates with {@code grpc-test.sandbox.googleapis.com}, which is a server
+ * managed by the gRPC team. For more information, see
+ * <a href="https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md">
+ *   Interoperability Test Case Descriptions</a>.
+ */
+@SuppressWarnings("serial")
+public final class HelloServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+    ManagedChannel channel =
+        ManagedChannelBuilder.forAddress("grpc-test.sandbox.googleapis.com", 443).build();
+    int desiredResponseSize = 12345;
+    TestServiceGrpc.TestServiceBlockingStub blockingStub = TestServiceGrpc.newBlockingStub(channel);
```

Comment:
Pointing new users at the interop service is very unfortunate. Is there no other API to use? On GAE getting credentials will be relatively easy.